simulators/seissol/v.0.2/readme.md

```diff
@@ -0,0 +1,86 @@
+# SeisSol Simulator
```

Comment:
This is great for end users as well as other developers. But I'm not sure of we should go so deep into it because it can slow down the release of new simulators. Regardless, great job.

Reply:
I have summarised the readme file to give a brief understanding about the compilation  seissol